expo-maps/src/NativeExpoMapView.tsx

```diff
@@ -8,8 +10,14 @@ import {
 
 export const NativeExpoGoogleMapsView = requireNativeViewManager(
   'ExpoGoogleMaps'
-) as React.FC<NativeExpoGoogleMapsViewProps>;
+) as React.ComponentType<NativeExpoGoogleMapsViewProps>;
 
 export const NativeExpoAppleMapsView = requireNativeViewManager(
   'ExpoAppleMaps'
-) as React.FC<NativeExpoAppleMapsViewProps>;
+) as React.ComponentType<NativeExpoAppleMapsViewProps>;
+
+export const NativeExpoAppleMapsModule = NativeModulesProxy.ExpoAppleMaps;
+export const NativeExpoGoogleMapsModule = NativeModulesProxy.ExpoGoogleMaps;
+
+export type NativeExpoAppleMapsViewType = typeof NativeExpoAppleMapsView;
+export type NativeExpoGoogleMapsViewType = typeof NativeExpoGoogleMapsView;
```

Comment:
Do we actually use these somewhere or we can remove them?

Reply:
I believe they can go ☺️